KBaseReport.html

```diff
@@ -1 +0,0 @@
<span class="primitive">int</span><span class="space"></span><span class="name">workspace_id</span><span>;</span></span></div><span class="tab"></span><span>}</span></span><span class="space"></span><span class="name" id="typedefKBaseReport.CreateExtendedReportParams">CreateExtendedReportParams</span><span>;</span></span></div><br><div><span class="funcdef"><div class="comment"><div><span class="tab"></span><span>/*</span></div><div><span class="tab"></span><span class="space"></span><span>*</span><span class="space"></span><span>* Create a report for the results of an app run. This method handles file</span></div><div><span class="tab"></span><span class="space"></span><span>*</span><span class="space"></span><span>* and HTML zipping, uploading, and linking as well as HTML rendering.</span></div><div><span class="tab"></span><span class="space"></span><span>*/</span></div></div><span class="tab"></span><span class="keyword">funcdef</span><span class="space"></span><span class="name" id="funcdefKBaseReport.create_extended_report">create_extended_report</span><span>(</span><span class="parameters"><span class="parameter"><span class="name"><a href="#typedefKBaseReport.CreateExtendedReportParams">CreateExtendedReportParams</a></span><span class="space"></span><span class="name">params</span></span></span><span>)</span><span class="space"></span><span class="keyword">returns</span><span>(</span><span class="returns"><span class="parameter"><span class="name"><a href="#typedefKBaseReport.ReportInfo">ReportInfo</a></span><span class="space"></span><span class="name">info</span></span></span><span>)</span><span class="space"></span><span class="keyword">authentication</span><span class="space"></span><span class="keyword">required</span><span>;</span></span></div><span>}</span><span>;</span></div><div class="index"><h2>Function Index</h2><div><span class="tab"></span><span class="name"><a href="#funcdefKBaseReport.create">create</a></span></div><div><span class="tab"></span><span class="name"><a href="#funcdefKBaseReport.create_extended_report">create_extended_report</a></span></div><h2>Type Index</h2><div><span class="tab"></span><span class="name"><a href="#typedefKBaseReport.CreateExtendedReportParams">CreateExtendedReportParams</a></span></div><div><span class="tab"></span><span class="name"><a href="#typedefKBaseReport.CreateParams">CreateParams</a></span></div><div><span class="tab"></span><span class="name"><a href="#typedefKBaseReport.File">File</a></span></div><div><span class="tab"></span><span class="name"><a href="#typedefKBaseReport.ReportInfo">ReportInfo</a></span></div><div><span class="tab"></span><span class="name"><a href="#typedefKBaseReport.SimpleReport">SimpleReport</a></span></div><div><span class="tab"></span><span class="name"><a href="#typedefKBaseReport.WorkspaceObject">WorkspaceObject</a></span></div><div><span class="tab"></span><span class="name"><a href="#typedefKBaseReport.ws_id">ws_id</a></span></div></div></body></html>
```

Comment:
I'd just remove this file completely

Reply:
I thought I'd got rid of it, but it keeps on returning like the bad penny.